TCAT/Cells/GeneralTableViewCell.swift

```diff
@@ -10,39 +10,53 @@ import UIKit
 
 class GeneralTableViewCell: UITableViewCell {
 
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
-    }
+    private let iconView = UIImageView()
+    private let titleLabel = UILabel()
 
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
-        super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
-    }
-
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-
-    override func layoutSubviews() {
-        super.layoutSubviews()
-
-        preservesSuperviewLayoutMargins = false
-        separatorInset = .zero
-        layoutMargins = .zero
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
 
         switch reuseIdentifier {
         case Constants.Cells.seeAllStopsIdentifier:
-            textLabel?.text = Constants.General.seeAllStops
-            imageView?.image = #imageLiteral(resourceName: "list")
+            titleLabel.text = Constants.General.seeAllStops
+            iconView.image = #imageLiteral(resourceName: "list")
             accessoryType = .disclosureIndicator
         case Constants.Cells.currentLocationIdentifier:
-            textLabel?.text = Constants.General.currentLocation
-            imageView?.image = #imageLiteral(resourceName: "location")
+            titleLabel.text = Constants.General.currentLocation
+            iconView.image = #imageLiteral(resourceName: "location")
```

Comment:
ik you didn't write this code but can we move this to a `func configure(for text: String, image: UIImage)` function instead. just b/c i want to standardize how we handle all cells and i don't think we usually do these checks on the reuseIdentifier. would be easier for people trying to read through the codebase in the future as well.